### PR TITLE
Add ProxyIOBufferSize option

### PR DESCRIPTION
**Pull request by jplindquist**
Add [ProxyIOBufferSize](https://httpd.apache.org/docs/2.4/mod/mod_proxy.html#proxyiobuffersize) for `mod_proxy`

Found we need to adjust this in combination w/ AJP to account for larger headers (cookies)

### CODE REVIEW

File: manifests/mod/proxy.pp [hunk 1]
```diff
@@ -1,33 +1,34 @@
...
 # @param proxy_via
 #   Set local IP address for outgoing proxy connections.
-# 
+#
 # @param proxy_timeout
 #   Network timeout for proxied requests.
-# 
+#
 # @see https://httpd.apache.org/docs/current/mod/mod_proxy.html for additional documentation.
 #
```
Comment (sheenaajay) [inline manifests/mod/proxy.pp:23]: Could you please add the new parameter information

File: manifests/mod/proxy.pp [hunk 2]
```diff
@@ -1,33 +1,34 @@
...
-  $package_name   = undef,
-  $proxy_via      = 'On',
-  $proxy_timeout  = undef,
+  $proxy_requests     = 'Off',
+  $allow_from         = undef,
+  $apache_version     = undef,
+  $package_name       = undef,
+  $proxy_via          = 'On',
+  $proxy_timeout      = undef,
+  $proxyiobuffersize  = undef,
```
Comment (sheenaajay) [inline manifests/mod/proxy.pp:31]: minor suggestion, can we keep the same naming format for the variable $proxy_iobuffersize

**Comment by sheenaajay** [comment]
Thanks @jplindquist for submitting the PR.

File: manifests/mod/proxy.pp (hunk 2 above)
Comment (jplindquist) [reply manifests/mod/proxy.pp]: Renamed to `proxy_iobuffersize` as suggested, thanks!

File: manifests/mod/proxy.pp (hunk 1 above)
Comment (jplindquist) [reply manifests/mod/proxy.pp:26]: Done

**Comment by jplindquist** [comment]
@sheenaajay The tests had previously passed, and after making the changes requested, there were a couple of failures?  Is that an issue, or is there a way to retry those, or something else I need to do to get that passing?

**Comment by sheenaajay** [comment]
Thank you @jplindquist for submitting the PR.

**Pull request merged**